util/cmake/ctd/configure_profile_properties.cmake

```diff
@@ -9,5 +9,13 @@ else ()
   set (CF_SEQAN_VERSION "${SEQAN_VERSION_STRING}")
 endif ()
 
+if (NOT CTD_PLUGIN_PACKAGE)
+  set (CTD_PLUGIN_PACKAGE "de.seqan")
```

Comment:
Just for making sure. :)
You said ".. not everyone is happy to name their package **seqan.de.*** ** just because they used seqan library." in the PR message.
It seems OK but just wish to check it.